src/moin/cli/maint/dump_html.py

```diff
@@ -254,26 +271,21 @@ def Dump(directory='HTML', theme='topside_cms', exclude_ns='userprofiles', user=
         logging.info("Dump html complete")
 
 
-def subitems(s, target='href="'):
+def get_used_dirs(query):
     """
-    fix links to subitems
-      * <a href="Home/subitem"> becomes <a href="Home(2f)subitem">
-      * do not change href="https://moinmo.in/FrontPage"
-      * do not change href="+serve/font_awesome/css/font-awesome.css"
-      * do not change href="_themes/topside_cms/css/theme.css"
+    get a list of item_names which have subitems (nodes in a tree)
     """
-    len_target = len(target)
-    idx = s.find(target)
-    while idx > 0:
-        idx2 = s.find('"', idx + 6)
-        assert idx2 > idx
-        sub = s[idx + len_target:idx2]
-        if sub and ('://' in sub or sub[0] in '+_#' or sub.startswith('static')):
-            idx = s.find(target, idx2)
-            continue
-        start = s[:idx + len_target]
-        end = s[idx2:]
-        sub = sub.replace('/', SLASH)
-        s = start + sub + end
-        idx = s.find(target, idx2)
-    return s
+    item = Item.create()  # gives toplevel index
+    # revs = flaskg.storage.search_meta(query, idx_name=LATEST_REVS, sortedby=NAME_EXACT, limit=None)
```

Comment:
this commented sort seems to be no longer needed, was added to prevent users.Home from being used as Home